www/views/partials/components/pricingEstimate.html

```diff
@@ -1,7 +1,7 @@
 <section class="pl-section" id="priceEstimate">
   <h3><a href="#priceEstimate">Price Estimate</a></h3>
   <p>When possible, show a price estimate on the left side of a Create form page so that customers can get an  understanding of how their actions impact potential costs. Changes in to settings in the create form (e.g. changing cpu count), should impact the values displayed in the price estimate.</p>
-  <p>Within a <code>.row</code>, use the <code>.priceEstimateContainer</code> to display the estimate values and <code>.createFormContainer</code> to contain the <code>&lt;form&gt;</code>.</p>
+  <p>Within a <code>.row</code>, use the <code>.priceEstimateContainer</code> to display the estimate values and <code>.createFormContainer</code> to contain the <code>&lt;form&gt;</code>. The price estimate container, by default, is positioned as <a href="utilities.html#sticky-positioning">sticky content</a>.</p>
```

Comment:
links need to be prefixed with `./` to get them to be rendered correctly in the dist output